pkg/util/constants.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+const (
+	DefaultOwner         = "rhdh-rhoai-bridge"
+	DefaultLifecycle     = "development"
+	StorageConfigMapName = "bac-import-model"
+	KeyQueryParam        = "key"
+	UpsertURI            = "/upsert"
+	CurrenKeySetURI      = "/currentkeyset"
```

Comment:
```suggestion
	CurrentKeySetURI      = "/currentkeyset"
```

(And elsewhere in the code to reflect this change)

Reply:
fixed everywhere via goland gloabl refactor / rename

part of next push